src/pytypes/onefuzztypes/models.py

```diff
@@ -568,6 +568,7 @@ class NodeCommandEnvelope(BaseModel):
 
 class Node(BaseModel):
     timestamp: Optional[datetime] = Field(alias="Timestamp")
+    initialized_at: Optional[datetime]  # Set only once when a node is intialized
```

Comment:
Instead of making this field optional, why not have it use `Field` to set a date if it doesn't already exist?

Reply:
This would change the meaning of the field. The idea is to record when we received the init message. A none value means that we have not received it yet. if we generate it automatically, we lose that information